src/com/second/collection/DoubleLinkedList.java

```diff
@@ -1,17 +1,51 @@
-package com.second.Model;
+package com.second.collection;
 
+import com.second.model.Student;
+
+import java.util.Iterator;
 import java.util.NoSuchElementException;
 
 public class DoubleLinkedList {
```

Comment:
Lusine jan what I was talking about to implement Iterable<Student> here as it only  has     Iterator<T> iterator(); abstract method